### ISSUE TITLE
OTP-13148 breaks parse transform [JIRA: RIAK-2555]

### ISSUE THREAD

**Issue by tsloughter** [basho/lager#357]
A change in the AST for OTP-19.x causes the parse transform to fail: 

```
error in parse transform 'lager_transform': {function_clause,
                                             [{lager_transform,
                                               '-walk_ast/2-fun-0-',
                                               [{typed_record_field,
                                                 {record_field,20,
                                                  {atom,20,vsn}},
                                                 {user_type,20,vsn,[]}}],
```

From OTP-13148: "This means that all parse transforms and all tools inspecting -record() declarations need to recognize {typed_record_field, Field, Type}."

**Comment by mrallen1** [comment]
Yeah there's already #321 which should fix this issue.

**Comment by tsloughter** [comment]
Oh, great. Will it be merged any time soon you think?

**Comment by mrallen1** [comment]
I hope so now that OTP 19 release is imminent.

**Comment by macintux** [comment]
Fixed in 2.2.3 and 3.2.1, now published.
